src/test/java/ioananadroflorea/tests/Tema5/HerokuAppTests.java

```diff
@@ -0,0 +1,65 @@
+package ioananadroflorea.tests.Tema5;
+
+import ioanaandroflorea.driver.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.List;
+
+public class HerokuAppTests {
+    public static void main(String[] args) {
+        goToBasicWebPageAndCheckElements();
+        elementAttributesExample();
+        locatorsFindByPage();
+        tableTestPage();
+    }
+
+    public static void goToBasicWebPageAndCheckElements() {
+        ChromeDriver driver = BrowserManager.createChromeDriver();
+        driver.get("https://testpages.herokuapp.com/styled/index.html");
+        WebElement firstLink = driver.findElement(By.className("basicpagetest"));
```

Comment:
Please re-check how this element is identified. I get an error locally.
![image](https://user-images.githubusercontent.com/18215854/211212205-2c856724-c001-4ee8-aad8-fb4e2d07e74a.png)